doc/enterprise-shield/p_enterprise_shield_config.md

```diff
@@ -0,0 +1,418 @@
+
+
+The following snippet shows the App1 Service configuration in the `gateway-config.xml` file.
+
+``` xml:
+<properties>
+  <name>dmz.backplane.hostname</name>
+  <value>gateway.example.dmz.net</value>
+</properties>
+
+<service>
+
+  <name>Internal App1</name>
+  <accept>wss://gateway.example.com:8443/app1</accept>
+  <connect>tcp://127.0.0.1:3101</connect>
+
+  <type>proxy</type>
+
+  <accept-options>
+    <http.transport>socks://${dmz.backplane.hostname}:1080</http.transport>
+    <socks.mode>reverse</socks.mode>
+    <socks.retry.maximum.interval>1 second</socks.retry.maximum.interval>
+    <socks.transport>
+      wsn+ssl://${dmz.backplane.hostname}:1080/shield
+    </socks.transport>  
+    <ws.inactivity.timeout>60 seconds</ws.inactivity.timeout>
+    <ssl.verify-client>required</ssl.verify-client>
+  </accept-options>
+
+  <cross-site-constraint>
+    <allow-origin>*</allow-origin>
+  </cross-site-constraint>
+
+</service>
+```
+
+The key configuration settings for the App1 service in the internal Gateway include the following elements:
+
+1.  Use a descriptive name in the [name](../admin-reference/r_configure_gateway_service.md#service) element for this service:
+
+     `                <name>Internal App1</name>          `
+
+    Use names that make it easier to identify the configuration. The example uses “Internal” in its name to differentiate this service from its corresponding service (named DMZ App1) in the DMZ Gateway.
+
+2.  Configure a secure connection from the client using the [accept](../admin-reference/r_configure_gateway_service.md#accept) element:
+
+     `                 <accept>wss://gateway.example.com:8443/app1</accept>           `
+
+    Use `wss:// scheme` for secure connections (and `ws:// scheme` for nonsecure connections) and replace the hostname and port with your hostname and port. Because all services typically use the same scheme, host, and port, you can differentiate between services by using a different path. The example uses `/app1` for its path. This is a logical path, and it is good practice to specify a path that is relevant to the service (such as `/stock`, `/prices`, `/traderApp`). See [Configuring Multiple Services on the Same Host and Port](../admin-reference/r_configure_gateway_multicast.md) for more information about using paths.
+
+3.  Connect the internal Gateway to the back-end service.
+
+    For some services, such as `proxy` or `amqp.proxy`, you configure the [connect](../admin-reference/r_configure_gateway_service.md#connect) element to connect to the back-end service used by the Gateway. Other services, such as JMS, use the [properties](../admin-reference/r_conf_jms.md) element to connect the internal Gateway to a JMS-compliant message broker.
+
+    The connect element can specify either the hostname or the IP address, and port of the back-end service:
```

Comment:
I this this should be the first part of this section. And what is currently the first paragraph should be moved lower. That's because this sentence is the crux of this section. The first paragraph is an addendum.

In fact, what is currently the first and third paragraphs can probably be reduced to something like:

> Most proxy services use the <connect> element to connect to the backend service. But others, like the JMS service, use the <properties> element instead. In that case, the host and port for the backend connection is contained in the <properties> element.